src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from "react-router-dom"
+import NavLogo from "../../Assets/nav-logo.png"
+
+const Navbar = () => {
+    return (
+        <nav class="navbar">
+            <div class="nav-logo">
+                <img src={NavLogo} alt="logo" />
+            </div>
+            <div class="nav-items">
+                <Link to="/" class="nav-link sm-title hide-md" href="./">About Us</Link>
```

Comment:
Suggestion: I don't think `href` attribute has any use here, correct me if I am wrong

Reply:
it has no use here. i didnt notice it thansk for pointng it out :)